contracts/proxy_to_vkr/Cargo.toml

```diff
@@ -20,17 +20,16 @@ crate-type = ["cdylib", "rlib"]
 backtraces = ["cosmwasm-std/backtraces"]
 
 [dependencies]
-cosmwasm-std = { version = "0.16.0" }
-cosmwasm-bignumber = "2.2.0"
-cw-storage-plus = "0.8.0"
+cosmwasm-std = "1.0"
+cw-storage-plus = "0.13.2"
 schemars = "0.8.1"
 serde = { version = "1.0.125", default-features = false, features = ["derive"] }
-thiserror = { version = "1.0.24" }
-cw2 = "0.8.0"
-cw20 = "0.8.0"
-valkyrie = { version = "1.0.7"}
-astroport = { version = "1.0.1" }
+thiserror = { version = "1.0.20" }
+cw2 = "0.13.2"
+cw20 = "0.13.2"
+astroport = { git = "https://github.com/astroport-fi/astroport-core.git", branch = "release/terra2" }
+valkyrie = { git = "https://github.com/valkyrieprotocol/contracts.git", branch = "feature/migrate_terra2" }
```

Comment:
I'm not sure that we can rely on temporal branch in 3rd party repos. Are we sure that Valkyrie team won't change/remove this branch?
Anyway I would suggest to use rev="..." to pin exact commit hash.